### PR TITLE
added hide windows update script

### PR DESCRIPTION
**Pull request by weastonjc**
## Issues
* [UN-1200](https://jumpcloud.atlassian.net/browse/<jira-id>) - Create script to block windows updates

## What does this solve?
This script allows a user to prevent a specific windows update from being downloaded to a machine.  The update that brought this issue to light was KB5012170 [blog explanation](https://www.neowin.net/news/microsoft-confirms-bitlocker-issues-in-the-latest-windows-updates/).

## Is there anything particularly tricky?
No

## How should this be tested?
**Note: This will need to be tested against Win 10 and 11.**
Verify that the update (in this case KB5012170) is not downloaded/installed on the machine.
The easiest  way to do this is the following:
- Go to Check for Updates (verify the target update is not pending) 
- Update History -> Uninstall Updates -> verify the update is not already installed.

Go to admin portal, run the _Windows - Exclude Update from Install_ script as a command against the target device.
- Verify the script completed successfully.
- On the target device, trigger it to check for updates.
- Verify that the target update is not downloaded or installed.

### CODE REVIEW

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md [hunk 1]
```diff
@@ -68,5 +68,5 @@ Downloads and installs the JumpCloud Remote Assist app on a Mac system.
 To import this command into your JumpCloud tenant run the below command using the [JumpCloud PowerShell Module](https://github.com/TheJumpCloud/support/wiki/Installing-the-JumpCloud-PowerShell-Module)
 
 ```
-Import-JCCommand -URL 'https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Mac%20Commands/Mac%20-%20Install%20Remote%20Assist.md'
+Import-JCCommand -URL "https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Mac%20Commands/Mac%20-%20Install%20Remote%20Assist.md"
```
Comment (dhaslemjc) [inline PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:71]: i take it this was an editor foo

**Review by dhaslemjc — APPROVED** [review]

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md (hunk 1 above)
Comment (weastonjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:71]: you got it!

File: PowerShell/JumpCloud Commands Gallery/commands.json [hunk 2]
```diff
@@ -97,6 +97,13 @@
...
     "description": "This command will download and install the CrowdStrike Falcon Agent to the device if it isn't already installed.\n\nIn order to use this command, follow the instructions from the [Installing the CrowdStrike Falcon Agent KB](https://support.jumpcloud.com/s/article/Installing-the-Crowdstrike-Falcon-Agent)\n\nSpecifically for this command:\n\n1. Download the CrowdStrike Falcon Agent installer and host it at a URL that your devices can access. Set this URL to the \"DownloadURL\" of this script.\n2. Extend the command timeout to a value that makes sense in your environment. The suggested command timeout for an environment with average network speeds on devices with average computing power is 10 minutes. Note that the command may timeout with a 124 error code in the command result window if not extended, but the script will continue to run.\n3. It is recommended to set the Launch Event for the command to \u201cRun As Repeating\u201d with an interval of one hour."
   },
+  {
+    "name": "Mac - Install Remote Assist | v1.0 JCCG",
```
Comment (rmulleyjc) [inline PowerShell/JumpCloud Commands Gallery/commands.json:101]: Are these additions intentional? All the remote assist stuff below too.

Comment (weastonjc) [reply PowerShell/JumpCloud Commands Gallery/commands.json:101]: this was an automated commit.  It looks like there is an automated workflow that updates this commands file [here](https://github.com/TheJumpCloud/support/blob/master/.circleci/workflows.yml#L316)

**Review by wwaughjc — APPROVED** [review]

File: PowerShell/JumpCloud Commands Gallery/commands.json (hunk 2 above)
Comment (wwaughjc) [reply PowerShell/JumpCloud Commands Gallery/commands.json:101]: We aren't ready to publish these commands just yet

File: PowerShell/JumpCloud Commands Gallery/commands.json [hunk 3]
```diff
@@ -272,6 +286,13 @@
...
+    "link": "https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Windows%20Commands/Windows%20-%20Install%20Remote%20Assist.md",
+    "description": "This command will download and install the JumpCloud Remote Assist app on a Windows device."
+  },
```
Comment (wwaughjc) [inline PowerShell/JumpCloud Commands Gallery/commands.json:295]:
```suggestion
```

File: PowerShell/JumpCloud Commands Gallery/commands.json [hunk 4]
```diff
@@ -97,6 +97,13 @@
...
+    "link": "https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Mac%20Commands/Mac%20-%20Install%20Remote%20Assist.md",
+    "description": "Downloads and installs the JumpCloud Remote Assist app on a Mac system."
+  },
```
Comment (wwaughjc) [inline PowerShell/JumpCloud Commands Gallery/commands.json:106]:
```suggestion
```

**Review by wwaughjc — CHANGES_REQUESTED** [review]

File: PowerShell/JumpCloud Commands Gallery/commands.json (hunk 2 above)
Comment (wwaughjc) [reply PowerShell/JumpCloud Commands Gallery/commands.json:101]: This isn't used just yet, so I think we're good to go.

**Review by wwaughjc — APPROVED** [review]
OK from the Remote Assist side of things

**Pull request closed without merging**